opencti-platform/opencti-front/playwright.config.ts

```diff
@@ -31,10 +31,11 @@ export default defineConfig({
         entryFilter: (entry) => true,
         sourceFilter: (sourcePath) => sourcePath.startsWith('src'),
       },
+      /*
```

Comment:
Are we keeping it for reference or should it be removed ?

Reply:
yes I'm keeping it in case we want to enable it again later, for now it's not used